### ISSUE TITLE
Build webpack failed

### ISSUE THREAD

**Issue by mymusise** [doccano/doccano#526]
When I run the command `docker-compose -f docker-compose.prod.yml up`,  it got a error with:
```
....
 ---> 2189196836cf
Step 5/6 : RUN npm install
 ---> Running in a15f3a271016
npm WARN deprecated core-js@2.6.11: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
npm WARN deprecated left-pad@1.3.0: use String.prototype.padStart()

> fibers@4.0.2 install /app/node_modules/fibers
> node build.js || nodejs build.js

gyp WARN install got an error, rolling back install
gyp ERR! configure error 
gyp ERR! stack Error: read ECONNRESET
gyp ERR! stack     at TLSWrap.onStreamRead (internal/stream_base_commons.js:201:27)
gyp ERR! System Linux 5.0.0-37-generic
gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild" "--release"
gyp ERR! cwd /app/node_modules/fibers
gyp ERR! node -v v13.1.0
gyp ERR! node-gyp -v v5.0.5
gyp ERR! not ok 
node-gyp exited with code: 1
Please make sure you are using a supported platform and node version. If you
would like to compile fibers on this machine please make sure you have setup your
build environment--
Windows + OS X instructions here: https://github.com/nodejs/node-gyp
Ubuntu users please run: `sudo apt-get install g++ build-essential`
RHEL users please run: `yum install gcc-c++` and `yum groupinstall 'Development Tools'` 
Alpine users please run: `sudo apk add python make g++`
gyp WARN install got an error, rolling back install
gyp ERR! configure error 
gyp ERR! stack Error: socket hang up
gyp ERR! stack     at connResetException (internal/errors.js:575:14)
gyp ERR! stack     at TLSSocket.socketOnEnd (_http_client.js:438:23)
gyp ERR! stack     at TLSSocket.emit (events.js:215:7)
gyp ERR! stack     at endReadableNT (_stream_readable.js:1199:12)
gyp ERR! stack     at processTicksAndRejections (internal/process/task_queues.js:80:21)
gyp ERR! System Linux 5.0.0-37-generic
gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild" "--release"
gyp ERR! cwd /app/node_modules/fibers
gyp ERR! node -v v13.1.0
gyp ERR! node-gyp -v v5.0.5
gyp ERR! not ok 
node-gyp exited with code: 1
Please make sure you are using a supported platform and node version. If you
would like to compile fibers on this machine please make sure you have setup your
build environment--
Windows + OS X instructions here: https://github.com/nodejs/node-gyp
Ubuntu users please run: `sudo apt-get install g++ build-essential`
RHEL users please run: `yum install gcc-c++` and `yum groupinstall 'Development Tools'` 
Alpine users please run: `sudo apk add python make g++`
npm WARN sass-loader@8.0.0 requires a peer of node-sass@^4.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN eslint-config-standard@14.1.0 requires a peer of eslint@>=6.2.2 but none is installed. You must install peer dependencies yourself.
npm WARN tsutils@3.17.1 requires a peer of typescript@>=2.8.0 || >= 3.2.0-dev || >= 3.3.0-dev || >= 3.4.0-dev || >= 3.5.0-dev || >= 3.6.0-dev || >= 3.6.0-beta || >= 3.7.0-dev || >= 3.7.0-beta but none is installed. You must install peer dependencies yourself.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.11 (node_modules/watchpack/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.11: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.11 (node_modules/nodemon/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.11: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.11 (node_modules/jest-haste-map/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.11: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.2 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! fibers@4.0.2 install: `node build.js || nodejs build.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the fibers@4.0.2 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-01-06T17_11_56_382Z-debug.log
ERROR: Service 'frontend' failed to build: The command '/bin/sh -c npm install' returned a non-zero code: 1
```

But it work on `v1.0.2`.

---------

*   Operating System: Ubuntu 18.04
*   doccano Version: master
*   When you install doccano: 2019/01/06
*   How did you install doccano (Heroku button etc):

**Comment by icoxfog417** [comment]
Thank you for reporting the issue. Can you install `v1.0.2` correctly (only occurs `v1.0.3`)?

**Comment by mymusise** [comment]
> Thank you for reporting the issue. Can you install `v1.0.2` correctly (only occurs `v1.0.3`)?

Yes, I can up the server in `v1.0.2` , but failed in `master (v1.0.3?)`

**Comment by manrajgrover** [comment]
Build fails for `v1.0.2` tag also.

```
Entrypoint app = 1a1c417ef28c37bcdf31.js f570f113bb041f7e42d2.js

ERROR in ./.nuxt/client.js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
Error: [BABEL] /app/.nuxt/client.js: Package exports for '/app/node_modules/@babel/helper-compilation-targets' do not define a '.' subpath (While processing: "/app/node_modules/@nuxt/babel-preset-app/src/index.js")
    at applyExports (internal/modules/cjs/loader.js:485:15)
    at resolveExports (internal/modules/cjs/loader.js:508:12)
    at Function.Module._findPath (internal/modules/cjs/loader.js:577:20)
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:879:27)
    at Function.Module._load (internal/modules/cjs/loader.js:785:27)
    at Module.require (internal/modules/cjs/loader.js:956:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/app/node_modules/@babel/preset-env/lib/debug.js:8:33)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1103:10)
    at Module.load (internal/modules/cjs/loader.js:914:32)
    at Function.Module._load (internal/modules/cjs/loader.js:822:14)
    at Module.require (internal/modules/cjs/loader.js:956:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/app/node_modules/@babel/preset-env/lib/index.js:11:14)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
 @ multi ./.nuxt/client.js app[0]

 FATAL  Nuxt build error

  at WebpackBundler.webpackCompile (node_modules/@nuxt/webpack/dist/webpack.js:5326:21)
  at processTicksAndRejections (internal/process/task_queues.js:93:5)
  at async WebpackBundler.build (node_modules/@nuxt/webpack/dist/webpack.js:5275:5)
  at async Builder.build (node_modules/@nuxt/builder/dist/builder.js:5598:5)
  at async Generator.initiate (node_modules/@nuxt/generator/dist/generator.js:70:7)
  at async Generator.generate (node_modules/@nuxt/generator/dist/generator.js:40:5)
  at async Object.run (node_modules/@nuxt/cli/dist/cli-build.js:96:7)
  at async NuxtCommand.run (node_modules/@nuxt/cli/dist/cli-command.js:2575:7)


   ╭─────────────────────────────╮
   │                             │
   │   ✖ Nuxt Fatal Error        │
   │                             │
   │   Error: Nuxt build error   │
   │                             │
   ╰─────────────────────────────╯

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! doccano@1.0.0 build: `nuxt build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the doccano@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-01-12T10_48_48_596Z-debug.log
ERROR: Service 'frontend' failed to build: The command '/bin/sh -c npm run build' returned a non-zero code: 1
```

**Comment by alokpadhi** [comment]
> > Thank you for reporting the issue. Can you install `v1.0.2` correctly (only occurs `v1.0.3`)?
> 
> Yes, I can up the server in `v1.0.2` , but failed in `master (v1.0.3?)`

How did you install with 'v1.0.2', is it working? I am also facing the similar problem.

**Comment by mymusise** [comment]
I just follow the document
```
$ git clone https://github.com/doccano/doccano.git
$ cd doccano
$ git checkout v1.0.2
$ docker-compose -f docker-compose.prod.yml up
```
Hope it help~

**Comment by mymusise** [comment]
> > > Thank you for reporting the issue. Can you install `v1.0.2` correctly (only occurs `v1.0.3`)?
> > 
> > 
> > Yes, I can up the server in `v1.0.2` , but failed in `master (v1.0.3?)`
> 
> How did you install with 'v1.0.2', is it working? I am also facing the similar problem.

I just follow the document
```
$ git clone https://github.com/doccano/doccano.git
$ cd doccano
$ git checkout v1.0.2
$ docker-compose -f docker-compose.prod.yml up
```
Hope it help~

**Comment by alokpadhi** [comment]
> > > > Thank you for reporting the issue. Can you install `v1.0.2` correctly (only occurs `v1.0.3`)?
> > > 
> > > 
> > > Yes, I can up the server in `v1.0.2` , but failed in `master (v1.0.3?)`
> > 
> > 
> > How did you install with 'v1.0.2', is it working? I am also facing the similar problem.
> 
> I just follow the document
> 
> ```
> $ git clone https://github.com/doccano/doccano.git
> $ cd doccano
> $ git checkout v1.0.2
> $ docker-compose -f docker-compose.prod.yml up
> ```
> 
> Hope it help~

It's still facing the same error. I tried on v1.0.1 also but still not working.

**Comment by dugarsumit** [comment]
You can fix this by changing the nodejs version from 13.1.0 to 13.6 in frontend/Dockerfile.

**Comment by alokpadhi** [comment]
> You can fix this by changing the nodejs version from 13.1.0 to 13.6 in frontend/Dockerfile.

I fix this by changing nodejs to 13.2.0 in frontend/Dockerfile. So I think any nodejs version above 13.1.0 works well.

**Comment by Hironsan** [comment]
This problem is fixed by https://github.com/doccano/doccano/pull/539.
Sorry for the inconvenience.
